pkg/flowkit/services/transactions.go

```diff
@@ -79,40 +79,92 @@ func (t *Transactions) GetStatus(
 	return tx, result, err
 }
 
+// TransactionAccounts define all the accounts for different transaction roles.
+//
+// If all roles are defined by the same account you can only set the payer.
+// You can read more about roles here: https://developers.flow.com/learn/concepts/accounts-and-keys
+type TransactionAccounts struct {
+	Proposer    *flowkit.Account
+	Authorizers []*flowkit.Account
+	Payer       *flowkit.Account
```

Comment:
I think here can be more than one payer ( for multisign )

Reply:
good idea

Reply:
but I will let the be part of another PR, we just need to add a method like `NewMultiSigTransactionAccounts`

Reply:
> I think here can be more than one proposer & payer ( for multisign )

I think there can be only one payer/proposer address, but it can have multiple required keys, so needs multi-sign.